### DISCUSSION TITLE
cannot import name 'tensor2vid'

### DISCUSSION THREAD

**Discussion by Psunshin** [huggingface/diffusers#9254] [Q&A]
how can i deal this question : ImportError: cannot import name 'tensor2vid' from 'diffusers.pipelines.text_to_video_synthesis.pipeline_text_to_video_synth'

**Comment by Psunshin** [comment]
please everybody help me

**Comment by tolgacangoz** [comment]
In the current `diffusers`, no `tensor2vid()` function exists in `diffusers.pipelines.text_to_video_synthesis.pipeline_text_to_video_synth`. So, this is expected. It was replaced with another function. Could you let me know how you encountered this error?

**Reply by Psunshin** [reply]
![1724641725122](https://github.com/user-attachments/assets/c5c5839f-e7e7-4592-a824-cb1a78094ea7)
from diffusers.pipelines.text_to_video_synthesis.pipeline_text_to_video_synth import tensor2vid, TextToVideoSDPipelineOutput
from diffusers.pipelines.stable_video_diffusion.pipeline_stable_video_diffusion import tensor2vid as svd_tensor2vid, StableVideoDiffusionPipelineOutput
from diffusers.loaders import LoraLoaderMixin, TextualInversionLoaderMixin

I don`t konw what i missed ,I can`t  read the code about tensor2vid

**Reply by tolgacangoz** [reply]
I guess you are using `alibaba/animate-anything` repo. Could you try to install its requirements in a way it says: https://github.com/alibaba/animate-anything#python-requirements
